lib/utils.py

```diff
@@ -14,13 +15,16 @@
 JSON = Dict[str, Any]
 
 BASE_URL = "https://mobile.southwest.com/api/"
-NTP_SERVER = "us.pool.ntp.org"
```

Comment:
Although this isn't directly relevant to the PR, this is a good change. Small and non-relevant changes would be best in a different PR so this one is easier to see exactly what changed to fix the 403s and 429s.

Reply:
Ill make this change in another PR.